k8s/cmd/references.md

```diff
@@ -16,7 +16,7 @@ drivers.
   -j, --dump-usage          Dump usage in JSON
   -g, --gen-doc             Generate reference docs, e.g., './references.md'
   -h, --help                help for vineyardctl
-      --kubeconfig string   kubeconfig path for the kubernetes cluster (default "/tmp/e2e-k8s.config")
+      --kubeconfig string   kubeconfig path for the kubernetes cluster (default "/opt/caoye/kubeconfig")
```

Comment:
Can you first

`export KUBECONFIG=$$HOME/.kube/config`

then run `./vineyardctl --gen-doc` to keep a nice look in the references? (Note the double `$` in the env above).

Reply:
My bad, I forget to update the kubeconfig. Thanks for pointing it out.